src/main/scala/nl/hugo/redbook/ch7/Par.scala

```diff
@@ -0,0 +1,114 @@
+package nl.hugo.redbook.ch7
+
+import java.util.concurrent._
+import language.implicitConversions
+
+object Par {
+  type Par[A] = ExecutorService => Future[A]
+
+  def run[A](s: ExecutorService)(a: Par[A]): Future[A] = a(s)
```

Comment:
OK, understood. I still fail to see the point of having `run(es)(a)` where `a(es)` would do the same.